server/planning/planning/module.py

```diff
@@ -24,3 +26,10 @@
     ),
     elastic=ElasticResourceConfig(),
 )
+
+planning_history_resource_config = ResourceConfig(
```

Comment:
I know it's not in the original, but we should probably have an index for the Item ID for this and Event history resources.

Example PlanningHistory index:
```python
mongo=MongoResourceConfig(
    indexes=[
        MongoIndexOptions(
            name="planning_id",
            keys=[("planning_id", 1)],
            unique=False
        ),
    ],
)
```